sample_code/ep143/rev01/t.py

```diff
@@ -0,0 +1,22 @@
+import subprocess
+import types
+
+import pytest
+
+
+@pytest.fixture
+def git_dir(tmpdir):
+    d = tmpdir.join('git')
+    subprocess.check_call(('git', 'init', str(d)))
+    commit_cmd = (
+        'git', '-C', f'{str(d)}', 'commit', '--allow-empty', '-m', 'init',
+    )
+    subprocess.check_call(commit_cmd)
```

Comment:
This line will still fail if the user doesn't have globally defined user.name and user.email :D
But whatever. I guess that it's good enough now.